packages/js/e2e-core-tests/specs/merchant/wp-admin-product-new.test.js

```diff
@@ -45,6 +45,12 @@ const runAddSimpleProductTest = () => {
 		});
 
 		it('can create simple virtual product and add it to the cart', async () => {
+
+			await page.setViewport( {
```

Comment:
For consistency, we should use `setBrowserViewport()` which can be imported from `@woocommerce/e2e-utils`.